### DISCUSSION TITLE
How should manage when execute user provided script with sending http reques?

### DISCUSSION THREAD

**Discussion by nphamvn** [sebastienros/jint#1765] [Q&A]
I want to execute user provided script. I want to allow user to create http request by providing a “plugin”. How should I manage the code execution?

**Comment by sebastienros** [comment]
You can expose a custom method, like `http_get` that will be using an HttpClient from dotnet. Another option is to provide an object `http` that will have multiple methods like `get(), post()` ... It's up to you. You can even pre-configure the http client instance to have limits, or resiliency.
